help/list-of-topics.md

```diff
@@ -0,0 +1,80 @@
+# List of topics in a channel
+
+You can view a list of recent topics any channel. It's a great way to get an
+overview if you like reading your messages one channel at a time.
```

Comment:
I think somehow this should say this is in the main center panel of the web app, as distinguished from the smaller left sidebar? It's a bit awkward because mobile works differently.

Reply:
Yeah, I dunno if there's a good way to put it, and I think the default assumption is that it's in the center panel.